Makefile

```diff
@@ -101,6 +97,19 @@ ifeq ($(ARCH),amd64)
 	docker build -f Dockerfile.$(ARCH) -t $(REGISTRY):$(TAG) .
 endif
 
+# This will build flannel natively using golang image
+dist/flanneld-e2e-$(TAG)-$(ARCH).docker:
+	# valid values for ARCH are [amd64 arm arm64 ppc64le s390x]
+	docker run -e GOARM=$(GOARM) \
```

Comment:
I'm not sure I understand why you've added this? What was wrong with the ` dist/flanneld-$(ARCH)` target?